fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/SearchFilter.java

```diff
@@ -42,6 +42,22 @@
         QueryType(String type) {
             this.type = type;
         }
+
+        /**
+         * Get the QueryType enum from it's search type.
+         *
+         * @param type  The type of query (i.e. "insensitive_contains")
+         *
+         * @return the enum QueryType
+         */
+        public static QueryType fromType(String type) {
```

Comment:
What's the reason for adding `final`? 
I've been told in previous PRs to remove `final` from parameters